src/storages/ConfigStorage.sol

```diff
@@ -653,6 +656,59 @@ contract ConfigStorage is IConfigStorage, OwnableUpgradeable {
     }
   }
 
+  function addStepMinProfitDuration(StepMinProfitDuration[] memory _stepMinProfitDurations) external onlyOwner {
+    uint256 length = _stepMinProfitDurations.length;
+    for (uint256 i = 0; i < length; ) {
+      if (_stepMinProfitDurations[i].fromSize >= _stepMinProfitDurations[i].toSize) revert IConfigStorage_BadArgs();
+      stepMinProfitDurations.push(_stepMinProfitDurations[i]);
+      emit LogSetStepMinProfitDuration(stepMinProfitDurations.length - 1, _stepMinProfitDurations[i]);
+      unchecked {
+        ++i;
+      }
+    }
+  }
+
+  function setStepMinProfitDuration(
+    uint256[] memory indexes,
+    StepMinProfitDuration[] memory _stepMinProfitDurations
+  ) external onlyOwner {
+    if (indexes.length != _stepMinProfitDurations.length) revert IConfigStorage_BadLen();
+    uint256 length = _stepMinProfitDurations.length;
+    for (uint256 i = 0; i < length; ) {
+      if (_stepMinProfitDurations[i].fromSize >= _stepMinProfitDurations[i].toSize) revert IConfigStorage_BadArgs();
+      stepMinProfitDurations[indexes[i]] = _stepMinProfitDurations[i];
+      emit LogSetStepMinProfitDuration(indexes[i], _stepMinProfitDurations[i]);
+      unchecked {
+        ++i;
+      }
+    }
+  }
+
+  function removeLastStepMinProfitDuration() external onlyOwner {
+    emit LogSetStepMinProfitDuration(
+      stepMinProfitDurations.length - 1,
+      IConfigStorage.StepMinProfitDuration({ fromSize: 0, toSize: 0, minProfitDuration: 0 })
+    );
+    stepMinProfitDurations.pop();
+  }
+
+  function getStepMinProfitDuration(uint256 marketIndex, uint256 sizeDelta) external view returns (uint256) {
+    uint256 length = stepMinProfitDurations.length;
+    if (length == 0) {
+      return minProfitDurations[marketIndex];
+    }
+    for (uint256 i = 0; i < length; ) {
+      if (sizeDelta >= stepMinProfitDurations[i].fromSize && sizeDelta < stepMinProfitDurations[i].toSize) {
+        // In-range
+        return stepMinProfitDurations[i].minProfitDuration;
+      }
+      unchecked {
+        ++i;
+      }
+    }
+    return 0;
```

Comment:
```suggestion
    return minProfitDurations[marketIndex];
```

to prevent wrong setting, when size delta exceed config range, we can using market duration as default

Reply:
ok